vendor/bat-native-ledger/src/bat/ledger/internal/wallet/wallet.cc

```diff
@@ -481,8 +481,8 @@ void Wallet::GetGrantViaSafetynetCheckCallback(const std::string& promotion_id,
     // Attestation failed
     braveledger_bat_helper::GRANT grant;
     grant.promotionId = promotion_id;
-    ledger_->OnGrantFinish(ledger::Result::SAFETYNET_ATTESTATION_FAILED,
-        grant);
+    //ledger_->OnGrantFinish(ledger::Result::SAFETYNET_ATTESTATION_FAILED,
+    //    grant);
```

Comment:
@NejcZdovc Any advice on how to handle this now that `OnGrantFinish` is replaced with a callback?